### PR TITLE
Gdr 1955

### PR DESCRIPTION
**Pull request by bczech**
# Description
## What changed?
Fix bug in calculating isobolograms
Related JIRA issue: GDR-1955

## Why was it changed?
To fix bug

# Checklist for sustainable code base
- [ ] I added tests for any code changed/added
- [ ] I added documentation for any code changed/added
- [ ] I made sure naming of any new functions is self-explanatory and consistent

# Logistic checklist
- [X] Package version bumped
- [X] Changelog updated

# Screenshots (optional)

### CODE REVIEW

File: R/combinations-isobolograms.R [hunk 1]
```diff
@@ -278,7 +277,8 @@ calculate_Loewe <- function(
             (df_iso_curve$log10_ratio_conc > 
                (df_iso_curve$log10_ratio_conc[x] - range / 2)) &
               (df_iso_curve$log10_ratio_conc <= 
-                 (df_iso_curve$log10_ratio_conc[x] + range / 2))]
+                 (df_iso_curve$log10_ratio_conc[x] + range / 2))],
+          na.rm = TRUE
```
Comment (gladkia) [inline R/combinations-isobolograms.R:281]: 👍

File: R/combinations-isobolograms.R [hunk 2]
```diff
@@ -342,7 +342,7 @@ get_isocutoffs <- function(df_mean, normalization_type) {
   if (min(df_mean[normalization_type == normalization_type, x], na.rm = TRUE) > 0.7) {
     iso_cutoffs <- NULL
   } else {
-    if (normalization_type == "GRvalue") {
+    if (normalization_type == "GR") {
```
Comment (gladkia) [inline R/combinations-isobolograms.R:345]: 👍

**Comment by gladkia** [comment]
Great job. Should some additional tests be added to protect against regression of this bug or maybe we have some tests coverting this bug already?

**Comment by bczech** [comment]
> Great job. Should some additional tests be added to protect against regression of this bug or maybe we have some tests coverting this bug already?

We should add test once our gDRtestData will be working properly, than we can extend our testing logic for combo assays as well

**Review by gladkia — APPROVED** [review]

**Pull request merged**